src/pages/blog/typography.md

```diff
@@ -75,18 +75,18 @@ But let's throw in a <b>tag</b>.
 
 ### Table 1: With Alignment
 
-| Tables        | Are           | Cool  |
-| ------------- |:-------------:| -----:|
+| Tables        |      Are      |  Cool |
+| ------------- | :-----------: | ----: |
 | col 3 is      | right-aligned | $1600 |
-| col 2 is      | centered      |   $12 |
-| zebra stripes | are neat      |    $1 |
+| col 2 is      |   centered    |   $12 |
+| zebra stripes |   are neat    |    $1 |
 
 ### Table 2: With Typography Elements
 
-Markdown | Less | Pretty
---- | --- | ---
-*Still* | `renders` | **nicely**
-1 | 2 | 3
+| Markdown | Less      | Pretty     |
```

Comment:
I know prettier or eslint probably did this but the markdown table here wasn't meant to be spaced out correctly. The point was to show that the table **still renders nicely** even if the spacing is off. I'll revert to the original.